web/src/popover_menus.js

```diff
@@ -705,4 +799,129 @@ export function initialize() {
             popover_instances.all_messages = undefined;
         },
     });
+
+    $("body").on("click", "#schedule-confirm", (e) => {
+        schedule_message_to_custom_date();
+
+        e.preventDefault();
+        e.stopPropagation();
+    });
+
+    function show_schedule_confirm_button(send_at_time, not_from_flatpickr) {
+        if (!send_at_time) {
+            return;
+        }
+        if (!not_from_flatpickr) {
+            selected_send_later_id = "send-later-custom-input";
+            send_at_time = format(new Date(send_at_time), "MMM d yyyy h:mm a");
+        }
+
+        selected_send_later_time = send_at_time;
+
+        $("#schedule-confirm").show();
+        tippy("#schedule-confirm", {
+            content: $t_html({defaultMessage: `Send at {send_at_time}`}, {send_at_time}),
+        });
```

Comment:
Does this clear an existing tooltip on the element? In any case, would it make sense for the API here to look more like setting a `data-scheduled-time` attribute on the button, and have a tooltip living in `tippy.js` read that attribute and display it?

Reply:
Yeah, this basically created multiple tippy instances for the same element, good this you noticed it.